dts/bindings/rtc/microcrystal,rv-8263-c8.yaml

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Daniel Kampert
+# Author: Daniel Kampert <DanielKampert@Kampis-Elektroecke.de>
+
+description: Micro Crystal RV-8263-C8 RTC
+
+compatible: "microcrystal,rv-8263-c8"
+
+include:
+  - name: rtc-device.yaml
+  - name: i2c-device.yaml
+
+properties:
+    int-gpios:
+      type: phandle-array
+      required: false
+
+    fast_mode:
+      type: boolean
+      required: false
+      description: |
+        Enable fast offset mode (Offset every 4 minutes).
+
+    offset:
+      type: int
+      required: false
```

Comment:
do not put required: false as it is redundant

Reply:
@decsny is it okay to amend these changes or should I push them in the normal way?

Reply:
you should amend the commits if that's what you mean